app/Jobs/SendMailJob.php

```diff
@@ -16,15 +17,15 @@ class SendMailJob implements ShouldQueue
 
     protected $mails;
 
-    public $tries = 5;
+    public int $tries = 5;
 
-    public function __construct($mailables)
+    public function __construct(ForgotPasswordMail $mailables)
```

Comment:
the $mailables attribute can be array or a Mailable interface, let's remove type from here